app/templates/base.html

```diff
@@ -94,7 +94,7 @@
         <div class="col">
              <div class="alert alert-info alert-dismissible fade show" role="alert">
                 <p class="mb-0">
-                    Aktualizace zdrojových dat:&nbsp;<strong>{{ last_update }}</strong> a webu:&nbsp;<strong>{{ now }}</strong>.
+                    Data aktualizována jen 1x denně - naposledy:&nbsp;<strong>{{ last_update }}</strong> a web:&nbsp;<strong>{{ now }}</strong>.
```

Comment:
What about this:
Zdrojová data jsou aktuálizována jen 1&times; denně, naposledy: xxx. Web byl aktualizován: xxx.